doc/2018/2018-09-04.md

```diff
@@ -0,0 +1,79 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight: Cosmic Cuttlefish (18.10) Feature Freeze Reminder
+
+We are nearing release for Cosmic (18.10) on October 18th. In preparation for the release,
+we just entered [Feature Freeze on August 23rd](https://lists.ubuntu.com/archives/ubuntu-devel-announce/2018-August/001248.html).
+Now is the time that we focus on bug fixes for our packages because most of the significant features
+targetting Cosmic are 'frozen'. Thank you for helping us build a better Ubuntu! We look forward to another great release.
+
+Please read the [Cosmic release schedule](https://wiki.ubuntu.com/CosmicCuttlefish/ReleaseSchedule) for more details about release-related dates.
+
+
+## cloud-init
+
+- Add unit tests for config/cc_ssh.py [Francis Ginther]
+- Fix the built-in cloudinit/tests/helpers:skipIf
+- read-version: enhance error message [Joshua Powers]
+- hyperv_reporting_handler: simplify threaded publisher
+- VMWare: Fix a network config bug in vm with static IPv4 and no gateway.
+  [Pengpeng Sun] ([LP: #1766538](https://bugs.launchpad.net/bugs/1766538))
+
+
+## Contact the Ubuntu Server team
+
+- Chat on #ubuntu-server on Freenode
+- Email the [ubuntu-server mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server)
+- Find us on the [Ubuntu Community Hub - server channel](https://discourse.ubuntu.com/c/server)
+
+## Bug Work and Triage
+
+- 268 in the [backlog]('https://bugs.launchpad.net/~ubuntu-server/+subscribedbugs)
+- 0 bugs reviewed since the last report
```

Comment:
Now I understand your question in chat better. The 0 number is not the # reviewed it is the backlog change +/-.

Reply:
We can either remove this line for this week or change it to "- no change to backlog since the last report"